test/integration/create-next-app/index.test.ts

```diff
@@ -473,6 +496,43 @@ describe('create next app', () => {
     })
   })
 
+  it('should use bun as the package manager on supplying --use-bun with example', async () => {
+    try {
+      await execa('bun', ['--version'])
+    } catch (_) {
+      // install bun if not available
+      await execa('curl', ['-fsSL', 'https://bun.sh/install | bash'])
```

Comment:
This also probably doesn't work :lolsob:

Reply:
@electroid is working on making bun available through npm though there will be some edgecases with the postinstall, similar to what esbuild & turbo run into

Reply:
Update: Bun is available on npm now

Reply:
Amazing!